avidemux_ja.ts

```diff
@@ -914,12 +914,12 @@ Drop timing informations ?</source>
     <message>
         <location line="+11"/>
         <source>Stat&amp;us bar</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
I would prefer to err on the side of caution and not use empty strings when a source string should remain untranslated. Not further delaying this PR, just as a note.